.travis.yml

```diff
@@ -2,7 +2,7 @@ language: node_js
 
 node_js:
   - "8"
-  - "9"
+  - "10"
```

Comment:
v9 is deprecated so switched to maintenance (8) and active (10):
https://nodejs.org/en/about/releases/

although not sure if we *need* (8) support any more?

Reply:
i say leave it as long as it doesn't become a hassle

Reply:
For me both options are right. But if no other people/projects are using it, we can remove.